ics-frontend/src/app/single-image-page/single-image-page.component.html

```diff
@@ -1,13 +1,16 @@
 <div class="container">
 <mat-card class="card">
-    <img mat-card-image src="https://material.angular.io/assets/img/examples/shiba2.jpg" alt="Photo of a Shiba Inu">
+    <img mat-card-image [src]="imageUrl" alt="Analysed image">
     <mat-card-content>
       <p>
-        get those labels (maybe the first 5)
+        <button mat-button *ngFor="let label of labels.slice(0, 5)">{{label}}</button>
```

Comment:
You can define the 5 label limit as a variable inside the component, so you can easily change it (if needed) from a single place

Reply:
adjusted